src/main/java/org/example/web/MainServlet.java

```diff
@@ -24,6 +26,12 @@ public class MainServlet extends HttpServlet {
     private final RequestParametersToArguments requestParametersToArguments = RequestParametersToArguments.getInstance();
     private final JsonToArgumentsConverter jsonToArgumentsConverter = JsonToArgumentsConverter.getInstance();
 
+    @Override
+    public void init() {
+        MainController.getInstance();
```

Comment:
Create separate method for the initialization of the service `MainController.init()`